notebooks/238-deeployd-if/deep-floyd-if.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
<p>Please provide more details about initial input (not clear how 64x64 image from previous stage become 2x6x256x256)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>added the following description:</p><p>"The second stage pipeline will use bilinear interpolation to upscale the 64x64 image that was generated in the previopus stage to a higher 256x256 resolution. Then it will denoise the image taking into account the encoded user prompt."</p>